modules/tinymce/src/core/main/ts/api/util/Delay.ts

```diff
@@ -129,7 +135,10 @@ const Delay: Delay = {
 
   /**
    * Sets an interval timer in ms and executes the specified callback at every interval of that time.
+   * <br>
+   * <em>`setInterval` has been deprecated and marked for removal in TinyMCE 6.0.</em>
```

Comment:
```suggestion
   * <em><code>setInterval</code> has been deprecated and marked for removal in TinyMCE 6.0. Use the native browser <code>setInterval</code> API instead.</em>
```
Same fixes as previously and we should also mention the replacement here.